build.gradle

```diff
@@ -53,6 +57,9 @@ allprojects {
 		maven {
 			url "https://oss.sonatype.org/service/local/repositories/releases/content"
 		}
+		maven {
```

Comment:
not needed anymore, released the confuse-io-json library

Reply:
Ok